dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -14,6 +14,8 @@
 bool CommandBarFlyoutTrace::s_IsDebugOutputEnabled{ false };
 bool CommandBarFlyoutTrace::s_IsVerboseDebugOutputEnabled{ false };
 
+static constexpr auto c_overlayCornerRadiusKey = L"OverlayCornerRadius"sv;
```

Comment:
this is not used anywhere anymore, you can remove it?:)

Reply:
ah yes good catch